bin/i18n-update.sh

```diff
@@ -80,25 +102,45 @@ fi
 # Get parameters
 PLUGINS=( "$@" )
 
+# Define constants
+TRANSLATIONS_OUTPUT_PATH="src/i18n-cache"
+
 echo -e "\n\033[1m== Updating i18n localizations ==\033[0m"
 
 # Validate parameters
 if [[ $((${#PLUGINS[@]}%2)) -ne 0 ]]; then
   error "Plugin arguments must be supplied as tuples (i.e. domain path/to/plugin)."
 fi
 
+# Check plugins parameters
 for (( index=0; index<${#PLUGINS[@]}; index+=2 )); do
+  PLUGIN_NAME=${PLUGINS[index]}
   PLUGIN_FOLDER=${PLUGINS[index+1]}
 
-  if [[ ! -d $PLUGIN_FOLDER ]]; then
-    NOT_FOUND_PLUGIN_FOLDERS+=( $PLUGIN_FOLDER )
-    echo -e "\033[0;31mPlugin folder \"$PLUGIN_FOLDER\" doesn't exist.\033[0m"
-  fi
+  check_plugin "$PLUGIN_NAME" "$PLUGIN_FOLDER"
 done
+# Check Gutenberg plugin
+check_plugin "gutenberg" "./gutenberg"
+
+# Stop if can't find any plugin folder
 if [[ $(arrayLength "${NOT_FOUND_PLUGIN_FOLDERS[@]+"${NOT_FOUND_PLUGIN_FOLDERS[@]}"}") -gt 0 ]]; then
   exit 1
 fi
 
+# Check if the process should be skipped due to the i18n cache presence
+if [[ -n ${SKIP_IF_CACHE:-} ]]; then
+  if [[ $(arrayLength "${NOT_FOUND_PLUGIN_I18N_CACHE[@]+"${NOT_FOUND_PLUGIN_I18N_CACHE[@]}"}") -eq 0 ]]; then
+    echo -e "-- Skipping the update due to SKIP_IF_CACHE enabled --"
+
+    # Even though localizations are not updated, we need to guarantee that i18n cache within react-native-editor package is updated
+    update_gutenberg_i18n_cache "$TRANSLATIONS_OUTPUT_PATH"
+
+    exit 0
+  else
+    echo -e "\n\033[1mForcing the update due to some cache folders not found.\033[0m"
+  fi
+fi
```

Comment:
We need to assure that we have translations in the cache folders (`src/i18n-cache`) in order to determine if the script should continue or not.

Reply:
This was addressed by adding the `i18n-check-cache` script ([reference](https://github.com/wordpress-mobile/gutenberg-mobile/blob/40fa7390b95a6d6db2b148ccadff7abc6e62ad9e/bin/i18n-check-cache.sh)).